index.php

```diff
@@ -9,10 +9,60 @@
     <div class="wrapper">
         <div id="logo-bar">
             <?php include 'logo-include.php'; ?>
-            <h1> Flock Information Database Home </h1>
-        </div>        
+            <h1> Home </h1>
+        </div>
+
         <div class="content">
-            <h2> Stick a login form here </h2>
+            <table id="register-login">
+            <tr>
+            <td><form method="post" action="index.php">
```

Comment:
Personally I don't really like double tagging when the inner portion is fairly long.
```html
<td>
    <form ...>
        ...
    </form>
</td>
```
Would be much better I think